sdk/src/main/java/com/tmsdurham/actions/ResponseBuilder.kt

```diff
@@ -361,6 +382,18 @@ data class BasicCard(internal var title: String = "",
         )
         return this
     }
+
+    /**
+     * Use one of the image display constants. If none is chosen,
+     * ImageDisplays.DEFAULT will be enforced.
+     *
+     * @param {ImageDisplayOption}
+     * @return {BasicCard} Returns current constructed BasicCard.
+     */
+    fun setImageDisplayOptions(imageDisplayOption: ImageDisplayOption): BasicCard {
```

Comment:
Same here - please use naming from the official SDK.  `setImageDisplay()`.  Also documentation is a bit different.  From the official SKD:
  /**
   * Sets the display options for the image in this Basic Card.
   * Use one of the image display constants. If none is chosen,
   * ImageDisplays.DEFAULT will be enforced.
   *